### PR TITLE
DDF-4240 Add Color styles to Multiline and MultiPolygon types

### PR DESCRIPTION
**Pull request by middlets719**
#### What does this PR do?
Adds feature styles for Multiline and MultiPolygon types
#### Who is reviewing it? 
@andrewkfiedler @bdeining @adimka @Schachte @djblue 
#### Select relevant component teams: 
@codice/ui 
#### How should this be tested?
Build UI and ensure Intrigue loads
#### What are the relevant tickets?
[DDF-4240](https://codice.atlassian.net/browse/DDF-4240)
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/openlayers/map.openlayers.js [hunk 1]
```diff
@@ -609,6 +609,23 @@ module.exports = function OpenlayersMap(
 
       feature.setId(locationModel.cid)
 
+      var styles = [
+        new Openlayers.style.Style({
```
Comment (bdeining) [inline ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/openlayers/map.openlayers.js]: Should we make the same changes to Cesium to keep them consistent?

File: ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/openlayers/map.openlayers.js [hunk 2]
```diff
@@ -609,6 +609,23 @@ module.exports = function OpenlayersMap(
 
       feature.setId(locationModel.cid)
 
+      var styles = [
```
Comment (willwill96) [inline ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/openlayers/map.openlayers.js]: ✏️ let

File: ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/openlayers/map.openlayers.js [hunk 3]
```diff
@@ -626,6 +643,23 @@ module.exports = function OpenlayersMap(
...
```
Comment (willwill96) [inline ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/openlayers/map.openlayers.js]: ✏️ same here

File: ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/openlayers/map.openlayers.js (hunk 2 above)
Comment (middlets719) [reply ui/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/openlayers/map.openlayers.js]: Will use const

**Comment by bdeining** [comment]
build now

**Comment by willwill96** [comment]
Hero Started

**Comment by willwill96** [comment]
🎉 Hero Successful 🎉 
1. Quickbuilt and installed.
2. Spot checking around UI for regressions
	- none found

**Comment by Schachte** [comment]
build now

**Pull request merged**